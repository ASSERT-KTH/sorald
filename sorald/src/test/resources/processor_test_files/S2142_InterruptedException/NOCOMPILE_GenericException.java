public class GenericException {
    public void run() {
        try {
            throw new InterruptedException();
        } catch (InterruptedException | Exception e) {

        }
    }
}
