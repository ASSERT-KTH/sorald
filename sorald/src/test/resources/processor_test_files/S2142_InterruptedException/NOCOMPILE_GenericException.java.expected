public class GenericException {
    public void run() {
        try {
            throw new InterruptedException();
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
        } catch (Exception e) {

        }
    }
}
